core/src/test/scala/org/apache/spark/util/JsonProtocolSuite.scala

```diff
@@ -904,6 +909,12 @@ private[spark] object JsonProtocolSuite extends Assertions {
     assertEquals(reason, newReason)
   }
 
+  private def testStackTrace(stackTrace: Array[StackTraceElement]): Unit = {
```

Comment:
Do you need a helper method for this if only used in one place? Not a big deal

Reply:
one helper method make code clean.